<!DOCTYPE html>
<html lang="en"><!-- InstanceBegin template="/Templates/base page w nav.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- InstanceBeginEditable name="doctitle" -->
<title>neuromics.research</title>
<!-- InstanceEndEditable -->
<!-- Bootstrap -->
<link href="css/bootstrap-4.4.1.css" rel="stylesheet">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;700&display=swap" rel="stylesheet">
<link href="css/style.css" rel="stylesheet" type="text/css">
<!-- InstanceBeginEditable name="head" -->
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">

<!-- Bootstrap -->
<link href="css/bootstrap-4.4.1.css" rel="stylesheet">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;700&display=swap" rel="stylesheet">
<link href="css/style.css" rel="stylesheet" type="text/css">
<!-- InstanceEndEditable -->
</head>
<body>
<div class="container mt-2 text-center"> <a href="index.html"> <img src="images/neuromics_banner.png" alt="BU neuromics" class="banner"></div>
</a>
<div class="container mt-2 text-center">
  <div class="row justify-content-center">
    <div class="col-2"><a href="index.html">home</a></div>
    <div class="col-2"><a href="people.html">people</a></div>
    <div class="col-2"><a href="research.html">research</a></div>
    <div class="col-2"><a href="projects.html">projects</a></div>
    <div class="col-2"><a href="other_stuff.html">other stuff</a></div>
  </div>
  <div class="col-12"> <img src="images/DNA_rule.png" alt="" width="90%"> </div>
</div>
<div class="container">
<main>
<div class="container text-center">
  <div class="row justify-content-center p-4">
    <h1 class="p-4"><!-- InstanceBeginEditable name="page title" -->research<!-- InstanceEndEditable --></h1>
    <div class="col-12 text-center"><!-- InstanceBeginEditable name="intro text" -->Our lab studies neurological/neurodegenerative disease, transcriptomics, and basic human genome questions using a combination of novel bioinformatic methodology, high-throughput sequencing data, high performance computing, and machine learning techniques.<!-- InstanceEndEditable --></div>
  </div>
  <!-- InstanceBeginEditable name="main" -->
      <div class="row justify-content-center">
		<div class="col-12 pb-4"><img src="images/DNA_rule.png" alt="" width="60%"></div>
		<div class="col-12"><h2>Neurodegenerative Disease</h2></div>
      </div>
      <div class="row justify-content-center align-items-center h-100 p-3">
        <div class="col-sm-10">
          <h3>Chronic Traumatic Encephalopathy Transcriptomics</h3>
          Chronic Traumatic Encephalopathy (CTE) is a progressive neurodegenerative disease caused by repetitive head impacts, as experienced by some military personnel, victims of domestic abuse, and players of violent sports. CTE pathology resembles that observed in Alzheimer's Disease, but has different spatial distribution and molecular signature. This area of research uses high throughput transcriptomic data generated from brains donated to the BU CTE Center Brain Bank to characterize and understand disease mechanisms. </div>
      </div>
      <div class="row justify-content-center align-items-center h-100 p-3">
        <div class="col-sm-10">
          <h3>Central Nervous System Microbiome</h3>
          Multiple lines of evidence suggest microbes play a direct role in central nervous system (CNS) disorders. Beyond the so-called “gut-brain” axis which posits interactions between gut microflora and the CNS influence neurological health, numerous studies have detected microbes directly in brain tissue from individuals with neurodegenerative disease, including Parkinson’s Disease, Alzheimer’s Disease, ALS, and multiple sclerosis. This area of research investigates the evidence of microbial signals found in transcriptomic data generated from CNS tissues across a variety of conditions.
		  </div>
      </div>
      <div class="row justify-content-center align-items-center h-100 p-3">
        <div class="col-sm-10">
          <h3>The Pan-Neurodegenerative Disease Phenotype</h3>
          Although the specific affected brain regions and pathology vary among different neurodegenerative diseases, gene expression studies in post mortem brain tissue show similar activation of neuroimmune and neuroinflammatory pathways. This area of research involves large scale comparison of post mortem brain gene expression data across many different neurodegenerative diseases to identify common and unique signals. </div>
      </div>
      <div class="row justify-content-center">
		<div class="col-12 pb-4"><img src="images/DNA_rule.png" alt="" width="60%"></div>
        <div class="col-12"><h2>Genomics Methods Development</h2></div>
      </div>
	  <div class="row justify-content-center align-items-center h-100 p-3">
        <div class="col-sm-10">
          <h3>Annotation-free Transcriptome Analysis</h3>
          Although only ~10% of the human genome codes for genes, >85% of the genome is transcribed to RNA. RNASeq data, especially ribo-depleted data, contain substantial numbers of reads that originate from intergenic regions that are typically ignored when performing gene-centric analyses using gene annotations. Our lab has developed methods that examine gene expression data for patterns ab initio, without filtering reads based on annotated regions. These methods have identified genomic loci that have transcript associated with biologically and clinically relevant features in Huntington's Disease that have gone unnoticed in annotation-restricted analyses.
		  </div>
      </div>
	  <div class="row justify-content-center align-items-center h-100 p-3">
        <div class="col-sm-10">
          <h3>Human Sequence Variation in Complex Genomic Loci</h3>
          Large scale public human genome sequence projects have generated individual-specific genome assemblies. These data provide exciting opportunities to identify and catalogue human variation in specific loci, especially those in complex, repetitive, or disease-associated parts of the genome. Our lab develops sequence analysis methodologies that investigate interesting regions of the human genome, including rDNA arrays, the MAPT region, and telomeres.
		  </div>
      </div>
			  <div class="row justify-content-center align-items-center h-100 p-3">
        <div class="col-sm-10">
          <h3>Metatranscriptomic Taxonomic and Functional Characterization</h3>
          Samples containing mixtures of organisms, including environmental, host microbiome site tissue, and symbiotic holobiont systems, contain a complex and diverse set of RNA transcripts. Due to dynamic organism and transcript copy number conditions and complex orthology among genomes, identifying which organisms and genomes are present in which proportion is challenging. Our lab develops methods that resolve ambiguity and mapping of RNA sequencing reads from mixed samples to taxonomic distributions and functional signatures with the goal of identifying associations with biological, environmental, and clinical variables of interest.
		  </div>
      </div>
	  <div class="row justify-content-center align-items-center h-100 p-3">
        <div class="col-sm-10">
          <h3>Sequence Graph Representations of Transcription</h3>
          Sequence graphs are data structures used to represent and encode variations within sets of related genomes. Our lab is interested in graph based representations of families of transcripts that can encode and reveal complex expression patterns associated with quantities of interest.
		  </div>
      </div>
    <!-- InstanceEndEditable -->
  </main>
</div>
<hr>
<footer class="text-center">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <p>Copyright or whatever © Adam Labadorf and Boston University. All rights reserved.</p>
      </div>
    </div>
  </div>
</footer>
<!-- jQuery (necessary for Bootstrap's JavaScript plugins) --> 
<script src="js/jquery-3.4.1.min.js"></script> 
<!-- Include all compiled plugins (below), or include individual files as needed --> 
<script src="js/popper.min.js"></script> 
<script src="js/bootstrap-4.4.1.js"></script>
</body>
<!-- InstanceEnd --></html>